src/stancjs/dune

```diff
@@ -1,7 +1,8 @@
 (executable
   (name stancjs)
   (libraries js_of_ocaml frontend stan_math_backend)
-  (preprocess (pps js_of_ocaml-ppx)))
+  (preprocess (pps js_of_ocaml-ppx))
+  (modes js))
```

Comment:
what does this modes js do? this breaks on my machine locally, but maybe I need to upgrade dune?

Reply:
I upgraded dune and it works now!